### DISCUSSION TITLE
Pake 可以添加打包二进制并且运行的功能。

### DISCUSSION THREAD

**Discussion by wangdashuaihenshuai** [tw93/Pake#324] [Ideas]
- 希望可以打包二进制，并且启动二进制，这样就可以运行一个后台，这样可以做的事情就更多了。
- 然后前端调用后端 API 来运行。

**Comment by tw93** [comment]
能详细说说吗？或者说类似的案例

**Reply by Tlntin** [reply]
他说的应该是打包本地二进制文件，让其成为后端api。
那么问题来了，前端在哪？如果没有前端，单独打包二进制很奇怪。
如果有前端，那么为啥不直接去用tauri，将二进制变成wasm，然后web调用wasm函数即可。

**Reply by tw93** [reply]
那应该是不太适合pake来做这个事情了

**Reply by wangdashuaihenshuai** [reply]
> 能详细说说吗？或者说类似的案例

这样就可以打包一个包含前端以及本地后端，不依靠网络的轻量app了。
